<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helix Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="VIEW_tree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1081</width>
      <height>661</height>
     </rect>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ACT_openDB"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="ACT_reload"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ACT_openDB">
   <property name="text">
    <string>Open database...</string>
   </property>
  </action>
  <action name="ACT_properties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Properties...</string>
   </property>
  </action>
  <action name="ACT_reload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
